pypowsybl/network.py

```diff
@@ -385,6 +404,8 @@ def create_ieee300() -> Network:
 def create_eurostag_tutorial_example1_network() -> Network:
     return Network(_pypowsybl.create_eurostag_tutorial_example1_network())
 
+def create_battery_network() -> Network:
```

Comment:
We should add this test network to the python API, just load and xiidm in the unit test